.github/workflows/terratest.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@44694675825211faa026b3c33043df3e48a5fa00 # v6.0.0
         with:
-          go-version: '1.22.x'
+          go-version: '1.25.x'
```

Comment:
alternatively we could set this to the `go.mod` file and let it use the version specified in there. This would reduce the number of places the information is stored (DRY)